gradle.properties

```diff
@@ -1,11 +1,11 @@
 group=io.ballerina.stdlib
 version=1.0.1-SNAPSHOT
```

Comment:
We need to bump to the next minor version since we are updating the lang version to `2201.8.0`

Reply:
We will bump the version of all stdlib modules to next minor version when we release the update 8.